.github/workflows/ci.yml

```diff
@@ -29,6 +29,6 @@ jobs:
       - name: Install Dependencies
         run: yarn install --pure-lockfile
       - name: Check Peer Dependencies
-        run: npx check-peer-dependencies
+        run: npx check-peer-dependencies --runOnlyOnRootDependencies
```

Comment:
> > hmm.. not sure what to do with the current error in the CI:
> > ```
> > npx check-peer-dependencies
> >   shell: /usr/bin/bash -e {0}
> >   ✅  ajv ^8.0.0 is required by ajv-formats@2.1.1 (8.11.0 is installed)
> >   ❌  ajv ^6.9.1 is required by ajv-keywords@[3](https://github.com/ui-router/angular/runs/7809448440?check_suite_focus=true#step:5:3).[5](https://github.com/ui-router/angular/runs/7809448440?check_suite_focus=true#step:5:6).2) (8.11.0 is installed)
> >   ✅  ajv ^8.8.2 is required by ajv-keywords@5.1.0 (8.11.0 is installed)
> > ```
> 
> For this issue. I noticed something was done to be fixed here [ui-router/sample-app-angular@3aad92b](https://github.com/ui-router/sample-app-angular/commit/3aad92bb4eeef00caf22a25bf3f04ab3b0c5bba3) Maybe try this same approach?

I added `--runOnlyOnRootDependencies` to the call to `check-peer-dependencies` instead, do you think this is not preferred and I should try the `ignore` approach?

Reply:
I'm not sure either but for consistency probably best to follow the same approach as it was done in the other project?

Reply:
okay, I can try to remove this and add that instead to see if it doesn't throw the error on `check-peer-dependencies`. I'll commit it or let you know if it doesn't work

Reply:
looks like it worked!
![Screen Shot 2022-09-15 at 11 35 25 AM](https://user-images.githubusercontent.com/1504590/190483167-f09cd1bc-2c09-47c2-b423-58e3fe0b88ea.png)